pugl/detail/mac_gl.m

```diff
@@ -91,6 +91,12 @@ - (void) update
 	[wrapper dispatchConfigure:[self bounds]];
 }
 
+- (void) drawRect:(NSRect)rect
+{
+	PuglWrapperView* wrapper = (PuglWrapperView*)[self superview];
+	[wrapper dispatchExpose:[self bounds]];
```

Comment:
The unused parameter warning here got me thinking: why bounds?  Did you choose this on purpose for some reason, rather than use the given rect?

Both seem to work fine here for me, though if there's a difference it would presumably be when subviews and such are involved...

Reply:
> why bounds? Did you choose this on purpose for some reason, rather than use the given rect?

An oversight... should be passing `rect` if it works.  Will test on my end in just a minute.

Reply:
Ok, tried the given `rect` works as expected. Change pushed to this P.R.